bluebottle/bluebottle_dashboard/static/jet/css/_dashboard.scss

```diff
@@ -187,6 +185,9 @@ ul.actionlist li {
     border-radius: 4px;
     margin-bottom: 20px;
     transition: background $transitions-duration;
+    .float-right {
```

Comment:
Better call it align-right. Float is counter intuitive

Reply:
I was re-using an existing one, but you're right. I will create a new class.